internal/pkg/logs/logs.go

```diff
@@ -0,0 +1,31 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
```suggestion
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0
```
It's actually the wrong header haha I fixed them in this PR: #1310

Reply:
🤦‍♂️